velox/substrait/SubstraitToVeloxExpr.cpp

```diff
@@ -52,39 +52,48 @@ SubstraitVeloxExprConverter::toVeloxExpr(
 
 std::shared_ptr<const core::ITypedExpr>
 SubstraitVeloxExprConverter::toVeloxExpr(
-    const ::substrait::Expression::ScalarFunction& sFunc,
+    const ::substrait::Expression::ScalarFunction& substraitFunc,
     const RowTypePtr& inputType) {
   std::vector<std::shared_ptr<const core::ITypedExpr>> params;
-  params.reserve(sFunc.args().size());
-  for (const auto& sArg : sFunc.args()) {
+  params.reserve(substraitFunc.args().size());
+  for (const auto& sArg : substraitFunc.args()) {
     params.emplace_back(toVeloxExpr(sArg, inputType));
   }
-  auto functionId = sFunc.function_reference();
-  auto veloxFunction =
-      substraitParser_.findVeloxFunction(functionMap_, functionId);
-  auto substraitType = substraitParser_.parseType(sFunc.output_type());
-  auto veloxType = toVeloxType(substraitType->type);
+  const auto& veloxFunction = substraitParser_.findVeloxFunction(
+      functionMap_, substraitFunc.function_reference());
+  const auto& veloxType = toVeloxType(
+      substraitParser_.parseType(substraitFunc.output_type())->type);
+
+  // Omit alias because name change is not needed.
+  if (veloxFunction == "alias") {
```

Comment:
I don't believe there is a function "alias" in Velox. What's the scenario where you hit this code path? Is there a test?

Reply:
SparkSql has "alias" function，there is an example
`wordCounts = textFile.select(explode(split(textFile.value, "\s+")).alias("word")).groupBy("word").count()` . So we add this code.